api/src/main/java/com/netflix/iceberg/transforms/Identity.java

```diff
@@ -17,9 +17,7 @@
 package com.netflix.iceberg.transforms;
 
 import com.google.common.base.Objects;
-import com.netflix.iceberg.expressions.BoundPredicate;
-import com.netflix.iceberg.expressions.Expressions;
-import com.netflix.iceberg.expressions.UnboundPredicate;
+import com.netflix.iceberg.expressions.*;
```

Comment:
Please don't use wildcard imports, it makes it difficult to see where symbols come from for readers.